http.c

```diff
@@ -490,6 +490,8 @@ static void
 evhttp_maybe_add_date_header(struct evkeyvalq *headers)
 {
 	if (evhttp_find_header(headers, "Date") == NULL) {
+		static const char *WEEKS[] = { "Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat" };
```

Comment:
(nginx awkwardly calls this a "week", a single week of days, but not plural weeks, and that isn't consistent with "months" just below)